Modules.md

```diff
@@ -140,6 +140,14 @@ The linear memory section may also contain an optional hint declaring the expect
 maximum heap usage. This hint is not semantically visible but can help a
 WebAssembly engine to optimize `grow_memory`.
 
+The linear memory section may optionally declare that the instance's
+linear memory is *externally aliasable*. How linear memory is aliased is up
+to the host environment (as with all module exports). The 
+[Web](Web.md#aliasing-linear-memory-from-JS) would reflect exported linear
+memory to JS as an `ArrayBuffer`. The MVP does not currently provide for
+*importing* linear memory though this may be added 
+[in the future](FutureFeatures.md#importing-linear-memory).
```

Comment:
I'm having trouble understanding your specific concern.  At the top you mention wasm+SAB interactions but then the rest of your comment seems to be general memory model concerns that are not specific to SAB.  There are some real concerns about resize+mprotect/SAB interactions outlined in Web.md, but those are quite different than what you're talking about here.  Moreover, we have agreed that, because threads (not just SAB interactions) are such a cross-cutting concern, that we should have a good experimental prototype up and working before v.1 is finished; that's #104.  So I'm not sure what specifically goes in a new issue.